Sentry.xcodeproj/project.pbxproj

```diff
@@ -5039,6 +5045,7 @@
 				LD_DYLIB_INSTALL_NAME = "$(DYLIB_INSTALL_NAME_BASE:standardizepath)/$(EXECUTABLE_PATH)";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				ONLY_ACTIVE_ARCH = YES;
+				OTHER_CFLAGS = "-DCARTHAGE_$(CARTHAGE)";
```

Comment:
Why `OTHER_CFLAGS` instead of `GCC_PREPROCESSOR_DEFINITIONS`?

Reply:
It was my first idea but it didn't work, which for me is weird.